### DISCUSSION TITLE
Add Editor Settings button to Project Manager window

### DISCUSSION THREAD

**Discussion by AriaMoKr** [godotengine/godot-proposals#6970] [Editor]
Currently, to change a setting that effects the editor you must first create/open project then go to editor settings.
For example on first launch you noticed that you want to change the default project location, you would need to first create a new project then go to editor settings to change this setting.
It might be handy to have the ability to change editor settings from the project manager window as well.

This may help address the following as well:
https://github.com/godotengine/godot-proposals/discussions/6259
https://github.com/godotengine/godot-proposals/discussions/4046
https://github.com/godotengine/godot-proposals/issues/1312
https://github.com/godotengine/godot-proposals/issues/3560

**Comment by theraot** [comment]
See https://github.com/godotengine/godot-proposals/issues/1602 - This looks like a duplicate to me.

**Reply by AriaMoKr** [reply]
agreed. I didn't see that one. thx
